templates/ListFacet.mustache

```diff
@@ -1,29 +1,44 @@
-{{#hasToggle}}
+{{#toggle}}
 	<div class="wikibase-faceted-search__facet-toggle">
-		<button class="cdx-button cdx-button--action-progressive cdx-button--weight-primary">
-			{{msg-and}}
-		</button>
-		<button class="cdx-button">
-			{{msg-or}}
-		</button>
+  {{#and}}
+    <button 
+      class="cdx-button {{#selected}}cdx-button--action-progressive cdx-button--weight-primary{{/selected}}"
+      {{#disabled}}disabled{{/disabled}}
+    >
+      {{label}}
+    </button>
+  {{/and}}
+  {{#or}}
+    <button
+      class="cdx-button {{#selected}}cdx-button--action-progressive cdx-button--weight-primary{{/selected}}"
+      {{#disabled}}disabled{{/disabled}}
+    >
+      {{label}}
+    </button>
+  {{/or}}
 	</div>
-{{/hasToggle}}
+{{/toggle}}
 
 {{#checkboxes}}
 	<div
-		class="wikibase-faceted-search__facet-item wikibase-faceted-search__facet-item-checkbox cdx-checkbox">
-		<div class="cdx-checkbox__wrapper">
-			<input
-				id="{{id}}"
-				class="cdx-checkbox__input"
-				type="checkbox"
-				{{#checked}}checked{{/checked}}
-			/>
-			<span class="cdx-checkbox__icon"></span>
-			<div class="cdx-checkbox__label cdx-label">
-				<label class="cdx-label__label" for="{{id}}">
-					<span class="cdx-label__label__text">{{>FacetItemLabel}}</span>
-				</label>
+		class="wikibase-faceted-search__facet-item wikibase-faceted-search__facet-item-checkbox cdx-checkbox"
+		data-value-id="{{value-id}}"
+	>
+    <div class="cdx-checkbox__wrapper">
+      <input
+        id="{{id}}"
+        class="cdx-checkbox__input"
+        type="checkbox"
+        {{#checked}}checked{{/checked}}
+        {{! Prevent input from being interacted by keyboard}}
+        tabindex="-1"
+      />
+      <span class="cdx-checkbox__icon"></span>
+      <div class="cdx-checkbox__label cdx-label">
+        <label class="cdx-label__label" for="{{id}}">
+          {{! Change to <div> tag instead of <span> tag for valid HTML }}
```

Comment:
I saw this comment before and do not know what I am supposed to take away from it

Reply:
It is caused by the previous merge, I have created a follow-up here: https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/104